### ISSUE TITLE
Issue with column_filters using python3

### ISSUE THREAD

**Issue by lostmarinero** [flask-admin/flask-admin#1029]
I am currently trying to set up views through flask-admin and I am having trouble using column filters. This is the error message I am receiving:

```sh
Traceback (most recent call last):
  File "devserver.py", line 13, in <module>
    app = create_app()
  File "/app/api/__init__.py", line 77, in create_app
    setup_admin(app, db_session)
  File "/app/api/admin.py", line 110, in setup_admin
    admin.add_view(TasksView(Tasks, db_session))
  File "/usr/local/lib/python3.4/dist-packages/flask_admin/contrib/sqla/view.py", line 289, in __init__
    menu_icon_value=menu_icon_value)
  File "/usr/local/lib/python3.4/dist-packages/flask_admin/model/base.py", line 584, in __init__
    self._refresh_cache()
  File "/usr/local/lib/python3.4/dist-packages/flask_admin/model/base.py", line 683, in _refresh_cache
    self._refresh_filters_cache()
  File "/usr/local/lib/python3.4/dist-packages/flask_admin/model/base.py", line 605, in _refresh_filters_cache
    self._filters = self.get_filters()
  File "/usr/local/lib/python3.4/dist-packages/flask_admin/model/base.py", line 826, in get_filters
    raise Exception('Unsupported filter type %s' % n)
Exception: Unsupported filter type name
```

Additionally, here is the code:

```python
class Tasks(CustomBase):
    __tablename__ = 'tasks'

    id = Column(UUIDType(binary=False), default=lambda: str(uuid.uuid4()), primary_key=True)
    name = Column(String(), default=None)
    details = Column(String(), default=None)
    due_date = Column(DateTime, default=None)


class TasksView(ModelView):
    session = db_session
    column_list = ('id', 'name', 'due_date', 'details'')
    column_filters = ('name', 'due_date', 'details')


def setup_admin(app, db_session):
    admin = Admin(app, index_view=AdminIndexView())
    admin.add_view(TasksView(Tasks, db_session))
 
    return admin

```

I am new to flask-admin, but was digging through the repo and found this component where if it isn't a type filter (even though the docs say you can use a string), then it calls self.scaffold_filters. However, self.scaffold_filters returns None always, causing the error to be raised.

```
    def get_filters(self):
        """
            Return a list of filter objects.
            If your model backend implementation does not support filters,
            override this method and return `None`.
        """
        if self.column_filters:
            collection = []

            for n in self.column_filters:
                if self.is_valid_filter(n):
                    collection.append(self.handle_filter(n))
                else:
                    flt = self.scaffold_filters(n)
                    if flt:
                        collection.extend(flt)
                    else:
                        raise Exception('Unsupported filter type %s' % n)
            return collection
        else:
            return None

    # Filter helpers
    def scaffold_filters(self, name):
        """
            Generate filter object for the given name
            :param name:
                Name of the field
        """
        return None

```

Any help is greatly appreciated

**Comment by pawl** [comment]
What does it say when you ```print(type(String()))```?

Should say: ```<class 'sqlalchemy.sql.sqltypes.String'>```

This worked for me (i'm using flask-sqlalchemy's db.String in that example): https://gist.github.com/pawl/ac76e8352fd1229c6411

**Comment by lostmarinero** [comment]
@pawl thanks for the response - I am using SQLAlchemy out of the box with the declarative base (not flask-sqlalchemy). When i ran `print(type(String()))` in the shell it gave me what you had, `<class 'sqlalchemy.sql.sqltypes.String'>`

Any other suggestions?

**Comment by alexleut** [comment]
I had the same issue. I fixed it by upgrading to flask-admin 1.3.0 from 1.1.0 and everything worked correctly (including scaffold_filter I assume.)
